standard.mk

```diff
@@ -27,11 +27,12 @@ OPERATOR_DOCKERFILE ?=build/ci-operator/Dockerfile
 
 BINFILE=build/_output/bin/$(OPERATOR_NAME)
 MAINPACKAGE=./cmd/manager
-GOENV=GOOS=linux GOARCH=amd64 CGO_ENABLED=0
-GOFLAGS=-gcflags="all=-trimpath=${GOPATH}" -asmflags="all=-trimpath=${GOPATH}" -mod=vendor
+unexport GOFLAGS
+GOENV=GOOS=linux GOARCH=amd64 CGO_ENABLED=0 GOFLAGS=
+GOBUILDFLAGS=-gcflags="all=-trimpath=${GOPATH}" -asmflags="all=-trimpath=${GOPATH}"
 export GO111MODULE=on
 
-TESTTARGETS := $(shell go list -e ./... | egrep -v "/(vendor)/")
+TESTTARGETS := $(shell go list -e ./...)
```

Comment:
Remove this line. Replace the var on L74 with the literal `./...`

Reply:
Done.